### DISCUSSION TITLE
Soft hyphens support in plate / rich-text editor

### DISCUSSION THREAD

**Discussion by Perni1984** [tinacms/tinacms#3173] [Help]
For a specific website we're experiencing long words breaking our layouts and the browsers built-in hyphenation support falling short. We would like to have the ability to insert soft hyphens / (`&shy;`) in the rich-text editor.

**Comment by kslstn** [comment]
+1! 

In the meanwhile, I'd like to have a way to prevent Tina CMS from removing `&shy;` from `rich-text` and `string` fields. It's quite tedious reinserting these every time Tina throws them out. Perhaps this already is possible?

**Reply by kslstn** [reply]
I was happy to see the first search result for my problem described it perfectly, but then I saw I've been here before ... Anybody out there in 2026?

**Comment by kulesy** [comment]
Hi, this will be investigated as part of #4503.
